elodie/filesystem.py

```diff
@@ -225,17 +226,42 @@ def process_file(self, _file, destination, media, **kwargs):
             shutil.move(_file, dest_path)
             os.utime(dest_path, (stat.st_atime, stat.st_mtime))
         else:
-            # Do not use copy2(), will have an issue when copying to a
-            # network/mounted drive using copy and manual
-            # set_date_from_filename gets the job done
-            shutil.copy(_file, dest_path)
+            # Do not use copy2(), will have an issue when copying to a network/mounted drive
+            # using copy and manual set_date_from_filename gets the job done
+            # shutil.copy(_file, dest_path)
+            
+            self.copyfile(_file, dest_path)
             self.set_utime(media)
 
+
         db.add_hash(checksum, dest_path)
         db.update_hash_db()
 
         return dest_path
 
+    def copyfile(self, src, dst):
```

Comment:
The build looked to have timed out in python3. I wonder if the proposed solution only works with python2.